scipy/special/basic.py

```diff
@@ -2126,6 +2126,53 @@ def factorialk(n, k, exact=True):
         raise NotImplementedError
 
 
+def voigt(x, sigma=1.0, gamma=1.0, mu=0.0):
+    r"""
+    Voigt profile.
+
+    The Voigt profile is a convolution of a 1D Normal distribution with
+    standard deviation ``sigma`` and a 1D Cauchy distribution with half-width at
+    half-maximum ``gamma``.
+
+    Parameters
+    ----------
+    x : array_like
+        Input argument.
+    sigma : float, optional
```

Comment:
Nit: `sigma` can be `array_like`, not necessarily float.

Reply:
I've just pushed the change to this.